### DISCUSSION TITLE
Using phone number to signin OTP

### DISCUSSION THREAD

**Discussion by MoathCodes** [saleor/saleor#11629] [Q&A] [answered]
Hi,
I would like to ask if there is any update on this topic I saw a couple of issues like #4633 [Issue](https://github.com/saleor/saleor/issues/4633) and #10522[ Discussion](https://github.com/saleor/saleor/discussions/10522) 
Is there a way to implement OTP in my app?

**Comment by timuric** [comment, marked as answer]
@IronDragonDev have you considered using custom auth? https://docs.saleor.io/docs/3.x/api-usage/authentication
You could use email field with a custom format, something like `123456@phone`
